ts/components/wallet/card/ActionIcon.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * Component that renders an icon from the
+ * icon font and wraps it with a touchable
+ * highlight (thus allowing onPress actions)
+ */
+import * as React from "react";
+import { View } from "react-native";
+import { StyleSheet, TouchableHighlight } from "react-native";
+import Icon from "../../../theme/font-icons/io-icon-font/index";
+import variables from "../../../theme/variables";
+
+type Props = Readonly<{
+  name: string;
+  onPress?: () => void;
+  size?: number;
+  touchable?: boolean;
+}>;
+
+const styles = StyleSheet.create({
+  iconWrapper: {
+    paddingLeft: 10
+  }
+});
+
+export default class ActionIcon extends React.Component<Props> {
+  public static defaultProps = {
```

Comment:
In this way you can put anything inside this object, also a property that is not in Props, use:

```
public static defaultProps: Partial<Props>
```